docs/dev_guide/contents/dependencies.rst

```diff
@@ -0,0 +1,20 @@
+.. _dependency_versions:
+
+*************************
+Dependency Support Policy
+*************************
+
+.. note::
+
+    This policy is based on `NEP-0029`_.
+
+sunpy has a short list of core dependencies (Python, numpy, astropy, parfive) and a long list of optional dependencies.
+The minimum version of these packages that we enforce follows this policy.
+
+* Python: Released in the prior 42 months from the anticipated release date.
+* astropy: Released in the prior 12 months from the anticipated release date.
+* Everything else: Released in the prior 24 months from the anticipated release date.
+
+Sponsored affiliated packages will support the sunpy LTS version at the time of their release.
```

Comment:
This makes it sound like they should support only this and not the .1 that follows?